src/main/java/org/jabref/gui/ai/components/util/notifications/NotificationComponent.java

```diff
@@ -0,0 +1,37 @@
+package org.jabref.gui.ai.components.util.notifications;
+
+import javafx.geometry.Insets;
+import javafx.scene.Node;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+
+import org.jabref.gui.icon.IconTheme;
+
+public class NotificationComponent extends HBox {
```

Comment:
Add some JavaDoc on the intention of this component.

Even better: Add some markdown file in `/docs` with screenshots. (Filed as https://github.com/InAnYan/jabref/issues/168)